masq_lib/src/constants.rs

```diff
@@ -12,6 +14,11 @@ pub const CURRENT_LOGFILE_NAME: &str = "MASQNode_rCURRENT.log";
 
 pub const MASQ_PROMPT: &str = "masq> ";
 
+pub const ETH_MAINNET_LABEL: &str = "eth";
+pub const ETH_ROPSTEN_LABEL: &str = "eth_t1";
```

Comment:
Technically this should probably be `eth_t3`... I'm all for avoiding underscores since Rust makes you vomit them all over the codebase like candy. For consistency with `"eth-mainent"` I recommend this be `eth-t3` or `eth-ropsten`